beacon-chain/rpc/eth/beacon/handlers_pool.go

```diff
@@ -335,28 +347,20 @@ func (s *Server) handleAttestationsElectra(ctx context.Context, data json.RawMes
 			failedBroadcasts = append(failedBroadcasts, strconv.Itoa(i))
 			continue
 		}
-		committeeIndex, err := att.GetCommitteeIndex()
-		if err != nil {
-			return nil, nil, errors.Wrap(err, "failed to retrieve attestation committee index")
-		}
-		subnet := corehelpers.ComputeSubnetFromCommitteeAndSlot(uint64(len(vals)), committeeIndex, att.Data.Slot)
+		subnet := corehelpers.ComputeSubnetFromCommitteeAndSlot(uint64(len(vals)), att.GetCommitteeIndex(), att.Data.Slot)
```

Comment:
Unrelated to this PR but calling `s.HeadFetcher.HeadValidatorsIndices(ctx, wantedEpoch)` allocate a new slice of active validator indices, it's expensive to just get the active validator count. `ActiveValidatorCount` is more suitable for this

Reply:
I will add this comment to my follow-up list